### ISSUE TITLE
Add some ways to "play" the game

### ISSUE THREAD

**Issue by shunjizhan** [UCSB-CS56-Projects/cs56-games-pokemon-map#20]
Right now except walking around the map, in no way our program is like a real game. We can add some goals in the game to make it playable.

~300 points

**Comment by mbahia** [comment]
Approved M.B.

**Comment by shunjizhan** [comment]
we added six events, one followed by the other.
There are six events in total in the game
1, after we walked in the flowers for some steps (around 20) then go to the grid behind the blue house, there will open a secret path through which we can go through the forest to a new map.
2, if we go into the big fish's mouth, the fish will create some lakes around the map, and there will be another secret path open (in the first map)
3, we need to go through the forest again and will find a secret path opened in the left down corner, we can go inside and when we reach the Pokemon ball, press "h" to interact.
4, then we will find many flowers were created, and another big Pokemon appeared in the town. We need to go around the map, reach that Pokemon, and interact with it, we will see it disappeared and left some red flowers.
5, go to the desert, where there were originally a big door and many big rocks, we will find the big Pokemon has came here. Interact with it, there will be another secret path opened, right beside us.
6, go with a secret path, through the lake and forest, we can finally interact with the big robot in the forest! after we interact with it, the map will become very beautiful, and that is the last event.
